scraper/src/Scraper.hs

```diff
@@ -12,11 +15,15 @@ runScrape = do
   case userOption of
     "1" -> putStrLn "Do the default flow"
     "2" -> putStrLn "Do the everything in database flow"
-    _   -> putStrLn "Oh god what happened"
+    _   -> putStrLn "Oh god what happened" --do the default flow anyway...?
 
-  putStrLn "Do a scrape job!"
+  scrape
   putStrLn "Save the results."
 
+scrape :: IO ()
+scrape = 
+  httpLBS "http://leg.colorado.gov/bill-search?field_sessions=10171&sort_bef_combine=field_bill_number%20ASC" >>= L8.putStrLn . getResponseBody
```

Comment:
Nice!

So we have a function `IO ByteString`. You can bind out of that to get a `ByteString`.

Next step we need is a function `ByteString -> Maybe Page` for some `data Page = ...`. This function will have many components:

- Identify the page that the web request returned.
- Scrape the valuable information out of the page.

This will end up being in multiple functions, most likely.